### PR TITLE
PAR stuart-gardner thomas

### PR DESCRIPTION
**Pull request by stuartogardner**

### CODE REVIEW

File: starter-code/package.json [hunk 1]
```diff
@@ -3,14 +3,17 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./bin/www"
+    "start": "nodemon ./bin/www"
```
Comment (yacinehmito) [inline starter-code/package.json]: As I said to Jad and Frederic: sneaky 😏 .

For the next time, try something like that:

```json
"scripts": {
    "start": "node ./bin/www",
    "dev": "nodemon ./bin/www"
}
```

Now you can do `npm run dev` to start nodemon, and you don't have to change the start script ;)

Comment (yacinehmito) [inline starter-code/package.json]: As I said to Jad and Frederic: sneaky 😏 .

For the next time, try something like that:

```json
"scripts": {
    "start": "node ./bin/www",
    "dev": "nodemon ./bin/www"
}
```

Now you can do `npm run dev` to start nodemon, and you don't have to change the start script ;)

File: starter-code/public/stylesheets/style.css [hunk 2]
```diff
@@ -6,3 +6,63 @@ body {
...
+  border: 1px solid #43a3e6;
+  border-radius: 4px;
+  color: #fff;
+  cursor: pointer;
+  display: inline-block;
+  font-size: 14px;
+  padding: 8px 16px;
+  text-decoration: none;
+  text-transform: uppercase;
+  transition: .3s ease background;
```
Comment (yacinehmito) [inline starter-code/public/stylesheets/style.css]: Awesome! ✨ 😮 ✨

File: starter-code/routes/auth-routes.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+
+authRoutes.get('/', (req, res, err) => res.render('index'))
+
+authRoutes.get("/signup", (req, res, next) => {
+  res.render("auth/signup");
+});
+
+authRoutes.post('/signup', (req, res, next) => {
+  let username = req.body.username;
```
Comment (yacinehmito) [inline starter-code/routes/auth-routes.js]: You can use `const` instead of `let` almost everywhere.

File: starter-code/routes/auth-routes.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+  let password = req.body.password;
+  let salt = bcrypt.genSaltSync(saltRounds);
+  let hashPass = bcrypt.hashSync(password, salt);
```
Comment (yacinehmito) [inline starter-code/routes/auth-routes.js]: Maybe you want to hash stuff only when you *know* you'll need to so.

File: starter-code/views/auth/login.ejs [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="stylesheets/style.css">
+  <title>Sign Up</title>
+</head>
+<body>
+  <form action="/login" method="POST" id="form">
+    <h2>Login</h2>
+
+    <label for="username">Username</label>
+    <input type="text" name="username" placeholder="what's yo name?" required>
```
Comment (yacinehmito) [inline starter-code/views/auth/login.ejs]: You can put an `input` tag *inside* a `label` tag to link them 😉 .

File: starter-code/views/auth/private.ejs [hunk 6]
```diff
@@ -0,0 +1,2 @@
+<h1>This page is private</h1>
+<img src="https://i.giphy.com/media/uEs5i50Syi8Du/source.gif" alt="here's some alt text">
```
Comment (yacinehmito) [inline starter-code/views/auth/private.ejs]: 🐧 🐧 ❤️

File: starter-code/routes/auth-routes.js (hunk 4 above)
Comment (liloow) [reply starter-code/routes/auth-routes.js]: That's indeed smarter 😁

File: starter-code/views/auth/private.ejs (hunk 6 above)
Comment (liloow) [reply starter-code/views/auth/private.ejs]: I knew you'd like it

**Pull request closed without merging**